src/learn.jl

```diff
@@ -32,7 +31,7 @@ Logs a series plot to `logger` under `field`, where...
 - `curves` is an iterable of the form `Tuple{Vector{Real},Vector{Real}}`, where each tuple contains `(x-values, y-values)`, as in the `Lighthouse.EvaluationRow` field `per_class_roc_curves`
 - `labels` is the class label for each curve, which defaults to the numeric index of each curve.
 """
-log_line_series!(logger, field::AbstractString, curves, labels=1:length(curves))
+log_line_series!(logger, field::AbstractString, curves; labels=1:length(curves))
```

Comment:
...is this intentional? Seems better, but also might break downstream consumers (although we suspect that that is exactly one consumer right now, so maybe not a big deal....)

Reply:
hmm in LWB we have a comma there:

```julia
"""
    Lighthouse.log_line_series!(::WandbLogger, field::AbstractString, curves, labels=nothing; kwargs...)
Log custom curves to wandb
Where...
    - `field::AbstractString` is the key to log the curves under
    - `curves` is an iterable of the form `Tuple{Vector{Real},Vector{Real}}`, where each tuple contains `(x-values, y-values)`, as in the `Lighthouse.EvaluationRow` field `per_class_roc_curves`
    - `labels` is the class label for each curve. If `nothing`, generates default labels that are the index of each curve
    - `kwargs` contains extra arguments for `wandb.plot.line_series`
"""
function Lighthouse.log_line_series!(::WandbLogger, field::AbstractString, curves,
                                     labels=nothing; kwargs...)
    return log_to_wandb!(field, make_wandb_line_series(curves, labels; kwargs...))
end
```

Reply:
Oops! I did not mean to change this in this PR, but I think it was supposed to be a comma... Not sure what happened 🤔

Reply:
not sure the pros and cons of having the labels as a positional or kw argument!

Reply:
ah i misunderstood!